playwright.config.ts

```diff
@@ -1,7 +1,40 @@
 import AxeBuilder from '@axe-core/playwright'
-import type { Expect, Locator, Page, PlaywrightTestConfig } from '@playwright/test'
+import type { Expect, Locator, Page, PlaywrightTestConfig, Project } from '@playwright/test'
 import { devices, expect } from '@playwright/test'
 
+import { findExamples } from './src/test-utils'
+
+const baseURL = 'http://localhost:8080'
+
+interface FrameworkProject extends Project {
+  grepInvert: RegExp
+}
+
+const frameworkProjects = (projects: FrameworkProject[]): Project[] => {
+  const reactComponents = findExamples('react', '.tsx').map(([component]) => component).join('|')
+
+  return projects.flatMap(({ grepInvert, ...project }) => [
+    {
+      ...project,
+      grepInvert: [grepInvert, /(@react)(\s|$)/],
+    },
+    {
+      ...project,
+      name: `react-${project.name}`,
+      testMatch: [
+        new RegExp(`(${reactComponents})/[\\w|-]+.spec.ts$`),
+        'accessibility.spec.ts',
+        'visual-regression.spec.ts',
+      ],
+      grepInvert: [grepInvert, /(@nunjucks)(\s|$)/].flat(),
+      use: {
+        ...project.use,
+        baseURL: new URL('/react', baseURL).toString(),
+      },
+    },
+  ])
+}
+
```

Comment:
This was fun. 
I added react projects based on the nunjuck project, 
This function ass prefixes react to the project name e.g `react-firefox` and will only test for the  React components that have been implemented. 

I've also added inverted greps for `@react` and `@nunjucks` see the `visual-regression.spec.ts` and `accesibility.spec.ts` for their usage